src/transform/FooterContainer.js

```diff
@@ -0,0 +1,85 @@
+import './FooterContainer.css'
+
+/**
+ * Ensures the 'Read more' section header can always be scrolled to the top of the screen.
+ * @param {!Document} document
+ * @param {!Window} window
+ * @return {void}
+ */
+const updateBottomPaddingToAllowReadMoreToScrollToTop = (document, window) => {
+  const div = document.getElementById('footer_container_ensure_can_scroll_to_top')
+  let currentPadding = parseInt(div.style.paddingBottom, 10)
+  if (isNaN(currentPadding)) { currentPadding = 0 }
+  const height = div.clientHeight - currentPadding
+  const newPadding = Math.max(0, window.innerHeight - height)
+  div.style.paddingBottom = `${newPadding}px`
+}
+
+/**
+ * Allows native code to adjust footer container margins without having to worry about
+ * implementation details.
+ * @param {!Document} document
+ * @param {!number} margin
+ * @return {void}
+ */
+const updateLeftAndRightMargin = (document, margin) => {
+  const elements = document.querySelectorAll(
+    '#footer_container_menu_heading, #footer_container_readmore, #footer_container_legal'
+  )
+  Array.from(elements)
```

Comment:
I see you already called this out in the todo but consider replacing `Array.from()` with `Array.prototype.slice.call()` for old time Android devices.